test/test.js

```diff
@@ -3994,6 +3994,40 @@
 
         deepEqual(actual, expecting);
       });
+
+      if (!/(Index|Key)/.test(methodName)) {
+        test('should enable shortcut fusion in chaining', 2, function() {
+          if (!isNpm) {
+            var callCount = 0;
+            var items = _.range(0, 251);
```

Comment:
Look at some of the other lazy sequence tests for their use of `LARGE_ARRAY_SIZE`.